packages/web-workers/src/webpack-parts/loader.ts

```diff
@@ -91,7 +91,7 @@ export function pitch(
 
   const workerOptions = {
     filename: addWorkerSubExtension(
-      compiler.options.output!.filename as string,
+      plugin.options.filename || (compiler.options.output!.filename as string),
```

Comment:
I think if they provide an explicit `filename`, we should not add the `.worker` to it — that part is up to them. Also, we can use `??` instead of `||` here

Reply:
👍 Fixed!